ci-operator/config/openshift/release/openshift-release-master__ci-4.16.yaml

```
@@ -591,6 +591,284 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-upgrade-vsphere
+- as: e2e-aws-ovn-capi
+  interval: 8h
+  steps:
+    cluster_profile: aws
+    env:
```

```suggestion
    env:
      AWS_PUBLIC_IPV4_POOL_ID: none
```
because capi-aws doesn't support BYOIPv4 yet.